tests/foreman/cli/test_vm_install_products_package.py

```diff
@@ -29,7 +29,6 @@
 from robottelo.constants import FAKE_0_CUSTOM_PACKAGE
 from robottelo.hosts import ContentHost
 from robottelo.products import DockerRepository
-from robottelo.products import PuppetRepository
```

Comment:
As puppet repo type is removed from 6.10, can't we remove `PuppetRepository` from `robottelo.products`?

Reply:
That is certainly something we can look into and perhaps something that @tstrych has already considered.

Reply:
The exam same code were already removed in #8927 
That PR's is waiting for @latran PR's. 
The `PuppetRepository` is removed in #8927 

If this is something important we can definitely merge it, or we can close it and wait once the mentioned PR is merged.
Hopefully soon.

Reply:
Going to merge this because of the very small scope. Thanks @tstrych for linking it to your consolidated work in #8927, that helps keep track of these changes. One fewer file for you to have to include as part of your commit.